docker-image/v1.3/debian-syslog/conf/fluent.conf

```diff
@@ -14,7 +14,7 @@
   port "#{ENV['SYSLOG_PORT']}"
   severity debug
   tag fluentd
-  protocol tcp
+  protocol "#{ENV['SYSLOG_PROTOCOL']}"
```

Comment:
Could you add `|| 'tcp'` for default value?

Reply:
Thanks @repeatedly, this has now been updated